test/test_train_imagenet.py

```diff
@@ -1,7 +1,50 @@
 import test_utils
 
+ALEXNET = 'alexnet'
```

Comment:
I think, since those have only one use, we can just inline the strings within the dictionary.
Removes quite a bit of code.